daemon/src/engine/qcommon/q_shared.cpp

```diff
@@ -1347,35 +1347,12 @@ Safe strncpy that ensures a trailing zero
 =============
 */
 
-#ifndef NDEBUG
```

Comment:
Yes, in this case we really have no use for the Hunk Debug, since we are trying to get rid of it